builds/checkin/mqtt.yaml

```diff
@@ -45,7 +45,7 @@ jobs:
         displayName: Build with no default features
       - bash: scripts/linux/generic-rust/build.sh --project-root "mqtt" --packages "mqttd/Cargo.toml" --manifest-path
         displayName: Build with default features
-      - bash: mqtt/build/linux/test.sh --report test-results.xml
+      - bash: mqtt/build/linux/test.sh -r --report test-results.xml
```

Comment:
Added to speed up tests (should build and run release) if this improves time to run and stops checkin from getting cancelled then I will open a new pr with just this change

Reply:
Seems to have improved time significantly! Created a pr for this small change, and if that goes in then can rebase this pr